web/help/rlt-notes-unit1.md

```diff
@@ -0,0 +1,16 @@
+Did you mean to reference a butterfly but not draw one?
+I understand your point. Just wondering if the student might be
+expecting a butterfly? Probably not if they understand polyline
+at this point but just making sure.
```

Comment:
I do see the possibility of confusion.  If you can explain the same idea and avoid that, go for it!

Reply:
Added a very simple butterfly. It's on the remote repo, but won't show up when using the preview link you sent. I am I doing something wrong?

 https://code.world/doc.html?path=https://raw.githubusercontent.com/Gravity-Well/codeworld/fiverr/web/help/GuideUnit1.md

Reply:
No, this is right.  Clickable code is the one thing that won't work in the preview URL.  That's because you are just looking at the guide, without the main CodeWorld window behind it.  Once I merge the pull request and students read it using the CodeWorld site, it will work.

If you really want them to see it inside the guide itself, then you can do that with an iframe.  You can search for `iframe` in the file, and copy one of those lines.  Replace the URL with the one you get by choosing "Share Without Code" from the Share dialog in CodeWorld.  That will actually run the code inside the Guide itself.

Reply:
As for this particular usage, I think this loses the point of the example.  The idea was to start with just the coordinate plane, then add points one by one.  If you just show them the completed program, I don't think it communicates that very well.

Reply:
Maybe if I reduced it to part of the butterfly. For example, part of the left wing. I'm just trying to make a bit more than an empty polyline. 
After showing part of the left wing, explain to them how they could remove the coordinate plane, view it, put it back in and add more. Thoughts

Reply:
I suspect the right thing to do here is to include multiple steps, and show how they can add to the program one vertex at a time.  It should probably be in a callout box if it gets that long, just to delineate it well from the rest of the text.

Reply:
I want to contribute in a meaningful way. I'd enjoy creating those steps if you think it would enhance the point. I think it would. But I can skip it and we can move on if you do not. Let's consider a video chat soon to discuss some of your ideas. The only real restrictions on my time are video calls I have on T,TH and F from 10:30-2:00 EST. Other than that, tasks can be shuffled.

Reply:
Yes, as I mentioned, I do think more detail here would be an improvement.  Just put it in a callout box so it doesn't interrupt the flow too much.  Will contact you elsewhere about setting up a video call.

Reply:
I do plan to do the incremental butterfly. Just did not include it in this push.